lib/streamlit/elements/arrow.py

```diff
@@ -86,6 +89,22 @@ def _arrow_dataframe(
             first, followed by 'col1', and will hide all other non-index columns. If
             None (default), the order is inherited from the original data structure.
 
+        column_config : dict or None
+            Configures how columns are displayed, e.g. their title, visibility, type, or
+            format. This needs to be a dictionary where each key is a column name and
+            the value is one of:
+
+            * ``None`` to hide the column.
+
+            * A string to set the display label of the column.
+
+            * One of the column types defined under ``st.column_config``, e.g.
+              ``st.column_config.NumberColumn(”Dollar values”, format=”$ %d”)`` to show
+              a column as dollar amounts. See more info on the available column types
+              and config options `here <TBD>`_.
```

Comment:
This TBD should be made sure that it's not in the doc strings but also I assume snehan will be looking at this?

Reply:
Yep, we don't have the URL for this yet, but we will update this as soon as we know the URL.

Reply:
This should be the URL: https://docs.streamlit.io/library/api-reference/data/st.column_config

Reply:
Updated 👍